packages/tools/pie-icons-webc/turbo.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": [
+    "//"
+  ],
+  "pipeline": {
+    "clean": {
+      "cache": false,
+      "dependsOn": []
+    },
+    "build:icons": {
+      "cache": false,
+      "dependsOn": [
+        "clean"
+      ],
+      "outputs": [
+        "icons"
+      ]
+    },
+    "build": {
+      "cache": false,
```

Comment:
Why not cache `pie-icons-webc` build scripts?

Reply:
This was something I was looking at. Hopefully resolved now.